src/pages/News/index.jsx

```diff
@@ -13,11 +13,11 @@ const News = () => {
         setColor(color === "green" ? "red" : "green");
     }
 
-    const {setNewsSources, setNewsInfo, allNewsInfo} = useStockNews();
+    const {setNewsSelections2, setNewsMasterlist2} = useStockNews();
 
     useEffect(() => {
-        setNewsSources();
-        setNewsInfo();
+        setNewsSelections2();
+        setNewsMasterlist2();
     }, [])
 
     //TODO: enable watchlist filter
```

Comment:
There are a few commented lines -                 
Spacer
Button mt={7} bg={color} onClick={handleChangeColor} watchlist only Button
                
Not sure if we need it in the codebase

Reply:
Perhaps they're needed for some testing? It's fine to leave them there for now imo and delete them later. It's not necessarily bad to have commented out code as long as it's removed from the final product.

Reply:
I left it there since we have the news page's watchlist filter as a stretch goal.

As Raghav says, I can just remove it from the final product if we don't end up implementing this feature. If this is bad practice though, I can remove it altogether from this PR, and if I need it later on, source it again from older commits

Reply:
Yep, it is not a very good practice to leave commented snippets of code in the codebase 
But I guess it is fine for now if that works best for you

Reply:
No that's alright I'll remove it and, as mentioned, pull it from an old commit if I need it in the future. Thanks :)